GNUmakefile

```diff
@@ -53,14 +53,12 @@ check-version:
 	scripts/check-version.sh "$(TAG)"
 
 .PHONY: openapi-pipeline
-openapi-pipeline: tools
+openapi-pipeline:
 	echo "Running OpenAPI Generation and Validation process"
 	$(MAKE) -C tools clean_client
 	echo "Running client generation"
 	$(MAKE) -C tools generate_client
 	echo "Validating generated SDK"
-	$(MAKE) v2-lint
-	$(MAKE) v2-test
```

Comment:
to follow some common patterns we've done across repos let's add a new target `v2-check` which is `v2-test v2-lint` somehwat strong opinion on that order to follow a red->green->refactor approach

Reply:
sounds good

Reply:
I'm removing those in favour of global test and lint